components/EditForm/index.js

```diff
@@ -80,7 +77,7 @@ export default function EditForm({ project, setEditMode }) {
       <StyledForm onSubmit={handleSubmit(onSubmit)}>
         <h2>Edit your project</h2>
         <StyledLabel>
-          Project title
+          <p>Project title</p>
```

Comment:
I just had to test it again myself with [caniunclude](https://caninclude.glitch.me/caninclude?child=p&parent=label). But it is not allowed to nest a paragraph element in a label element.